architecture/topics/cisco.adoc

```diff
@@ -0,0 +1,18 @@
+////
+Module included in the following assembly:
+
+* architecture/networking/network_plugins.adoc
+////
+
+The Cisco ACI CNI plug-in for {product-title} provides integration between the APIC controller and one or more {product-title} clusters connected to a Cisco ACI fabric.
+
+This integration is implemented across two main functional areas:
+
+. The Cisco ACI CNI plugin extends the ACI fabric capabilities to {product-title} clusters in order to provide IP address management, networking, load balancing, and security functions for {product-title} workloads. The Cisco ACI CNI plugin connects all {product-title} Pods to the integrated VXLAN overlay provided by Cisco ACI.
+
+. The Cisco ACI CNI plug-in models the entire {product-title} cluster as a VMM domain on the Cisco APIC. This provides APIC with access to the inventory of resources of the {product-title} cluster, including the number of {product-title} nodes, {product-title} namespaces, services, deployments, Pods, their IP and MAC addresses, interfaces they are using, and so on. APIC uses this information to automatically correlate physical and virtual resources in order to simplify operations.
+
+The Cisco ACI CNI plug-in is designed to be transparent for {product-title} developers and administrators and to integrate seamlessly from an operational standpoint.
+
+For more information, see 
+link:https://www.cisco.com/c/en/us/td/docs/switches/datacenter/aci/apic/white_papers/Cisco-ACI-CNI-Plugin-for-OpenShift-Architecture-and-Design-Guide.html[Cisco ACI CNI Plugin for Red Hat OpenShift Container Platform Architecture and Design Guide].
```

Comment:
s/Plugin/Plug-in

Reply:
"Plugin" here is in the title and link of the third-party doc, so I'm inclined to leave their spelling in place here.